hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/dispatcher/Dispatcher.java

```diff
@@ -33,25 +39,23 @@
  */
 public class Dispatcher {
 
+    // TODO: Intermediate solution until we find a better way to get the service-key
+    private static final String TOKEN_SERVICE_KEY = new TokenServiceImpl().getServiceName();
```

Comment:
@rbair23 already started to provide static names in the service interfaces. Don't know if it is already in the develop branch.

Reply:
Nope, it is part of this PR: https://github.com/hashgraph/hedera-services/pull/4696

Reply:
Maybe you can adopt the pattern.